package/yast2-schema.spec

```diff
@@ -26,16 +26,61 @@ Source0:        %{name}-%{version}.tar.bz2
 Group:	        System/YaST
 License:        GPL-2.0+
 
-# FIXME: drop yast2-all-packages some day
-BuildRequires:	trang yast2-all-packages
-
-# not covered by yast2-all-packages
-BuildRequires:	yast2-online-update-configuration
-
-
-# yast2-core omited from some reason in buildservice, adding explicitly
-BuildRequires:  yast2-core
-#!BuildIgnore: yast2-build-test yast2-online-update
+Url:            https://github.com/yast/yast-schema
+
+# Dependencies needed to build the package
+BuildRequires:	trang yast2-devtools yast2-testsuite
+
+# All packages providing RNG files for AutoYaST
+# in /usr/share/YaST2/schema/autoyast/rng/*.rng
+BuildRequires: autoyast2
+BuildRequires: yast2
+BuildRequires: yast2-add-on
+BuildRequires: yast2-audit-laf
+BuildRequires: yast2-auth-client
+BuildRequires: yast2-auth-server
+BuildRequires: yast2-bootloader
+BuildRequires: yast2-ca-management
+BuildRequires: yast2-country
+BuildRequires: yast2-dhcp-server
+BuildRequires: yast2-dns-server
+BuildRequires: yast2-firewall
+BuildRequires: yast2-firstboot
+BuildRequires: yast2-ftp-server
+BuildRequires: yast2-http-server
+BuildRequires: yast2-inetd
+BuildRequires: yast2-installation
+BuildRequires: yast2-iscsi-client
+BuildRequires: yast2-kdump
+BuildRequires: yast2-mail
+BuildRequires: yast2-network
+BuildRequires: yast2-nfs-client
+BuildRequires: yast2-nfs-server
+BuildRequires: yast2-nis-client
+BuildRequires: yast2-nis-server
+BuildRequires: yast2-ntp-client
+BuildRequires: yast2-online-update-configuration
+BuildRequires: yast2-printer
+BuildRequires: yast2-proxy
+# Available on SLE (12) only, needs to be adjusted for next SLE
+%if 0%{?suse_version} == 1315
+BuildRequires: yast2-registration
```

Comment:
No, no testcase. I've added that to All Ideas on Yast Team Trello. I know this might be a problem in the future and I've seen some discussions on ML that `this is not a problem at all` TM. Plus the fact that my team don't care either... saying `I told you` is not enough :)

On the other hand, I regret more that we cannot check for `sle_version` or `is_this_sles?`